javascript/builtins/RegExp.json

```
@@ -426,6 +426,57 @@
             }
           }
         },
+        "hasIndices": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/RegExp/hasIndices",
+            "support": {
+              "chrome": {
+                "version_added": "91"
+              },
+              "chrome_android": {
+                "version_added": "91"
+              },
+              "edge": {
+                "version_added": false
```

It also works in Edge 90

```suggestion
                "version_added": "90"
```

---

Done.